src/devtools/client/debugger/src/actions/sources/breakableLines.js

```diff
@@ -4,28 +4,28 @@
 
 //
 
-import { getSourceActorsForSource, getBreakableLines } from "../../selectors";
-import { setBreakpointPositions } from "../breakpoints/breakpointPositions";
-import { loadSourceActorBreakableLines } from "../source-actors";
+import { getSourceActorsForSource } from "../../selectors";
+import {
+  loadSourceActorBreakableLines,
+  loadSourceActorBreakpointHitCounts,
+} from "../source-actors";
 
-function calculateBreakableLines(positions) {
```

Comment:
It looks a little weird to be replacing this function with a function that does a totally different thing, but that's just a trick of the diff - this was unused code.